content/python/concepts/math-module/terms/math-ceil/math-ceil.md

```diff
@@ -20,7 +20,7 @@ Takes in a numeric data type and rounds the value up to the nearest integer.
 math.ceil(n)
 ```
 
-The `math.ceil()` function takes in a value and returns its ceiling, the smallest integer greater than or equal to that value.
+The **`math.ceil()`** function takes in a value and returns its ceiling, the smallest integer greater than or equal to that value.
 
 ## Example 1
```

Comment:
```suggestion
## Example 
```

Reply:
I would also suggest removing one of the existing examples from these entries. These are all generally pretty basic methods and there isn't a lot gained by showing essentially the same functionality multiple times.